style.css

```diff
@@ -17,12 +17,37 @@ body {
     border-right: 1px solid rgba(0, 0, 0, 0.25);
   }
 
+  .rainbow {
+    font-family: sans-serif;
+    background: linear-gradient(to right, #ef5350, #f48fb1, #7e57c2, #2196f3, #26c6da, #43a047, #eeff41, #f9a825, #ff5722);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  }
+
+  .footer {
+    position: absolute;
+    bottom: 0;
+    background-color: orange;
+    width: 100%;
+    height: 4.5rem;

```

Comment:
The footer is over the last element of the side bar

Reply:
fixed : )